### ISSUE TITLE
integration with threejs

### ISSUE THREAD

**Issue by dberardo-com** [Leaflet/Leaflet#9203]
### Checklist

- [X] I've searched through the [plugins](https://leafletjs.com/plugins.html) to make sure this feature isn't already available, or think it shouldn't require a plugin.
- [X] I've searched through the [current issues](https://github.com/Leaflet/Leaflet/issues) to make sure this feature hasn't been requested already.
- [X] I agree to follow the [Code of Conduct](https://github.com/Leaflet/Leaflet/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.

### Motivation

basically i am trying to reproduce this behavior using leaflet: https://docs.mapbox.com/mapbox-gl-js/example/add-3d-model-threebox/

### Suggested solution

the only "guidelines" i am aware of are highlighted here: https://stackoverflow.com/questions/59733455/location-based-ar-with-three-js-react-camera-configuration

**Comment by jonkoops** [comment]
We don't have any integrations with Three.js, Leaflet is a 2D mapping library. To my knowledge no such plugins exist.
